docs/src/pages/lab/breadcrumbs/CollapsedBreadcrumbs.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Breadcrumbs from '@material-ui/lab/Breadcrumbs';
+import Breadcrumb from '@material-ui/lab/Breadcrumb';
+import Link from '@material-ui/core/Link';
+
+const styles = theme => ({
+  root: {
+    justifyContent: 'center',
+    flexWrap: 'wrap',
+  },
+  paper: {
+    padding: `${theme.spacing.unit}px ${theme.spacing.unit * 2}px`,
+  },
+});
+
+function handleClick(event) {
+  event.preventDefault();
+  alert('You clicked a breadcrumb.'); // eslint-disable-line no-alert
+}
+
+function CollapsedBreadcrumbs(props) {
+  const { classes } = props;
+
+  return (
+    <Paper className={classes.paper}>
+      <Breadcrumbs maxItems={2} arial-label="Breadcrumb navigation">
+        <Breadcrumb>
+          <Link color="inherit" href="#" onClick={handleClick}>
```

Comment:
Do we need both `onClick` and ‘`href`here?

Reply:
Yes, otherwise the page scroll to the top.

Reply:
Ah, okay. I didn't spot the `event.preventDefault();` (on mobile without reading glasses! :).

This isn't how we've shown `href` in other demos. I think it's confusing, as it isn't how it would be used typically in production.